app/services/cycle_timetable.rb

```diff
@@ -95,64 +109,64 @@ def self.current_cycle_schedule
     SiteSetting.cycle_schedule
   end
 
-  def self.schedules
+  def self.schedules(year = current_year)
     {
-      real: CYCLE_DATES[CURRENT_YEAR_FOR_SCHEDULE],
+      real: CYCLE_DATES[year],
 
       today_is_mid_cycle: {
         apply_1_deadline: 1.day.from_now.to_date,
         apply_2_deadline: 3.days.from_now.to_date,
         find_closes: 4.days.from_now.to_date,
-        find_reopens: 5.days.from_now.to_date,
-        apply_reopens: 6.days.from_now.to_date,
+        find_opens: 5.days.from_now.to_date,
+        apply_opens: 6.days.from_now.to_date,
       },
 
       today_is_after_apply_1_deadline_passed: {
         apply_1_deadline: 1.day.ago.to_date,
         apply_2_deadline: 2.days.from_now.to_date,
         find_closes: 3.days.from_now.to_date,
-        find_reopens: 4.days.from_now.to_date,
-        apply_reopens: 5.days.from_now.to_date,
+        find_opens: 4.days.from_now.to_date,
+        apply_opens: 5.days.from_now.to_date,
       },
 
       today_is_after_full_course_deadline_passed: {
         apply_1_deadline: 2.days.ago.to_date,
         apply_2_deadline: 1.day.from_now.to_date,
         find_closes: 2.days.from_now.to_date,
-        find_reopens: 3.days.from_now.to_date,
-        apply_reopens: 4.days.from_now.to_date,
+        find_opens: 3.days.from_now.to_date,
+        apply_opens: 4.days.from_now.to_date,
       },
 
       today_is_after_apply_2_deadline_passed: {
         apply_1_deadline: 3.days.ago.to_date,
         apply_2_deadline: 1.day.ago.to_date,
         find_closes: 1.day.from_now.to_date,
-        find_reopens: 2.days.from_now.to_date,
-        apply_reopens: 3.days.from_now.to_date,
+        find_opens: 2.days.from_now.to_date,
+        apply_opens: 3.days.from_now.to_date,
       },
 
       today_is_after_find_closes: {
         apply_1_deadline: 4.days.ago.to_date,
         apply_2_deadline: 2.days.ago.to_date,
         find_closes: 1.day.ago.to_date,
-        find_reopens: 1.day.from_now.to_date,
-        apply_reopens: 2.days.from_now.to_date,
+        find_opens: 1.day.from_now.to_date,
+        apply_opens: 2.days.from_now.to_date,
       },
 
       today_is_after_find_reopens: {
```

Comment:
Is this name is a little confusing now? Should it just be `today_is_after_find_opens`?